docs/services/qix-engine/apis/file-loading/file-connector-api.md

```diff
@@ -0,0 +1,382 @@
+
+<!-- markdownlint-disable -->
+<!-- proselint-disable -->
+# gRPC File Connector API
+
+_gRPC File Connector API for version 12.350.0._
+
+Package: **qlik.filehosting**
```

Comment:
filehosting/file connector/file streaming. Is there alignment on the naming, and if so, can we ensure the code is aligned too? If we're introducing a new API we should make sure it's consistent in naming too.

Reply:
I brought up the question a while back and then we went for the `gRPC File Connector API` to align it with the `gRPC Data Connector API`. Might be worth to take the discussion one more iteration :)